samples/README.md

```diff
@@ -0,0 +1,20 @@
+# How to run the samples
+
+1. Clone this repo and navigate to the `snippets` directory.
+   ```shell
+   git clone https://github.com/googlepis/java-pubsub
+   cd java-pubsub/samples/snippets
+   ```
+
+1. With [Maven] installed and the environment variable `GOOGLE_APPLICATION_CREDENTIALS` set to point to your [GCP service account] key file, run a sample that has the to-do section completed:
```

Comment:
Is this still the right thing to ask?  We don't really want them downloading SA's any more.

Reply:
Good call. Updated.